packages/cli/src/models/files/ZosPackageFile.ts

```diff
@@ -17,6 +18,11 @@ export default class ZosPackageFile {
     publish: boolean;
   };
 
+  public static getZosversion(fileName: string = 'zos.json'): string | null {
+    const file = fs.parseJsonIfExists(fileName);
+    return file.zosversion || null;
+  }
```

Comment:
I think we should move the `checkVersion` thing outside the constructor, I imagine that's why you needed to put such behavior as a static method. Probably, we can move to a "before each" statement for any command.

Reply:
Yeap... I didn't want to change the actual behavior of how we are checking `1.x` and `2` versions. OTOH, This is the only way that came into my mind to get the `zosversion` *before* initializing a `ZosPackageFile` inside the scripts functions.